src/main/java/com/linkedin/kmf/services/ProduceService.java

```diff
@@ -238,7 +238,7 @@ public boolean isRunning() {
     private final ConcurrentMap<Integer, Boolean> _produceErrorInLastSendPerPartition;
     private final Map<String, String> _tags;
 
-    public ProduceMetrics(final Metrics metrics, final Map<String, String> tags) {
+    ProduceMetrics(final Metrics metrics, final Map<String, String> tags) {
```

Comment:
Since ProduceService implements App external users may have code that instantiates it.

Reply:
Amended for external clients.